libbeat/cmd/instance/beat.go

```diff
@@ -1466,3 +1470,17 @@ func sanitizeIPs(ips []string) []string {
 	}
 	return validIPs
 }
+
+func mergeOutputQueueSettings(bc *beatConfig) error {
```

Comment:
Is "merge" the right verb here? It has me thinking that I can put settings in both places and they will be combined.

Reply:
changed verb to "promote".  I _think_ that is a better description since we are moving the queue settings "up" in the config file.  But I'm open to other suggestions